script/main.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-use-before-define */
+
+const myLibrary = [];
+
+const addBookButton = document.getElementById('add');
+const info = document.getElementsByClassName('book-input');
+const form = document.getElementsByTagName('form')[0];
+const cards = document.getElementById('cards');
+
+let bookCounter = 0;
+let bookFormOn = 0;
+
+function Book(title, author, pages, id) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.id = id;
+}
+
+function clearCards() {
+  while (cards.firstChild) {
+    cards.removeChild(cards.firstChild);
+  }
+}
+
+function deleteCard(event) {
+  const index = event.target.value;
+  myLibrary.splice(index, 1);
+  viewBooks();
+}
+
+function read(event) {
+  const index = event.target.value;
+  myLibrary[index].status = true;
+  event.target.setAttribute('class', 'form-hide');
+}
+
+function renderCard(book, index) {
+  const divCard = document.createElement('div');
+
+  divCard.setAttribute('class', 'book-card');
+  divCard.setAttribute('id', book.id);
+  cards.appendChild(divCard);
+  const h2 = document.createElement('h2');
+  h2.textContent = book.title;
+
+  const h3 = document.createElement('h3');
+  h3.textContent = `Author:${book.author}`;
+
+  const p = document.createElement('p');
+  p.textContent = `Total pages:${book.pages}`;
+
+  const b1 = document.createElement('button');
+  b1.textContent = 'Delete';
+  b1.setAttribute('value', index);
+  b1.addEventListener('click', deleteCard);
+  b1.setAttribute('class', 'delete button');
+  const b2 = document.createElement('button');
+  b2.addEventListener('click', read);
+  b2.textContent = 'Read';
+  b2.setAttribute('class', 'read button');
+  b2.setAttribute('value', index);
+  divCard.appendChild(h2);
+  divCard.appendChild(h3);
+  divCard.appendChild(p);
+  divCard.appendChild(b1);
+  divCard.appendChild(b2);
+}
+
+function viewBooks() {
+  clearCards();
+  for (let i = 0; i < myLibrary.length; i += 1) {
+    renderCard(myLibrary[i], i);
+  }
```

Comment:
Seems like you are rendering card by card. You might consider building all the cards in an array and then append that once to the DOM.

<details>
 <summary>Hint</summary>
 
```js
function renderCard(book, index) {
  const divCard = document.createElement('div');

  divCard.setAttribute('class', 'book-card');
  divCard.setAttribute('id', book.id);

  // Let's not append one by one, this is expensive
  // cards.appendChild(divCard);

  const h2 = document.createElement('h2');
  h2.textContent = book.title;

  const h3 = document.createElement('h3');
  h3.textContent = `Author:${book.author}`;

  const p = document.createElement('p');
  p.textContent = `Total pages:${book.pages}`;

  const b1 = document.createElement('button');
  b1.textContent = 'Delete';
  b1.setAttribute('value', index);
  b1.addEventListener('click', deleteCard);
  b1.setAttribute('class', 'delete button');
  const b2 = document.createElement('button');
  b2.addEventListener('click', read);
  b2.textContent = 'Read';
  b2.setAttribute('class', 'read button');
  b2.setAttribute('value', index);
  divCard.appendChild(h2);
  divCard.appendChild(h3);
  divCard.appendChild(p);
  divCard.appendChild(b1);
  divCard.appendChild(b2);

  // Let's return the card created so we can use it to append all at once.
  return divCard;
}

function viewBooks() {
  clearCards();
  cards.append(...myLibrary.map(renderCard));
}
```
</details>